rero_ils/modules/loans/jsonschemas/loans/loan-ils-v0.0.1.json

```diff
@@ -50,10 +50,10 @@
       "type": "string",
       "title": "Request pickup location ID"
     },
-    "request_expire_date": {
+    "request_expiry_date": {
```

Comment:
I do not know if this will break `invenio_circulation` or not. The code of `invenio_circulation`  rely on 'request_expire_date'

Reply:
In `invenio-circulation` this string is to be find only in the schemas and the mappings.

https://github.com/inveniosoftware/invenio-circulation/search?q=request_expire_date&unscoped_q=request_expire_date